x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils.test.ts

```diff
@@ -258,7 +256,7 @@ describe.each([
 
   describe('transformFindAlerts', () => {
```

Comment:
Should we add a test here for when actions are passed in? Saw we're doing that now in find_rules_route.

Reply:
Good good catch. I will add a test here.